### PR TITLE
Add get employee by id endpoint

### PR DESCRIPTION
**Pull request by mstruckus**

### CODE REVIEW

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -47,5 +49,26 @@ public async Task<IActionResult> Register([FromBody] RegisterRequest model)
...
         }
+
+        [HttpGet]
+        [Route("employees/{userId}")]
+        public async Task<IActionResult> GetEmployeeById(string employeeID)
+        {
+            try
+            {
+                var result = await _employeeManager.GetEmployeeById(employeeID);
+                var response = new EmployeeInfo()
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/Controllers/EmployeesController.cs]: I would suggest to project DTO in manager, but when I think about it more, I think its kind of optional. What is your opinion?

Comment (tomasmuzas) [reply BackendApartmentReservation/BackendApartmentReservation/Controllers/EmployeesController.cs]: And wait, why are you creating the same `EmployeeInfo` type, just without id? It is a must to return an ID

File: BackendApartmentReservation/BackendApartmentReservation/Repositories/EmployeeRepository.cs [hunk 2]
```diff
@@ -28,6 +28,11 @@ public async Task<DbEmployee> GetEmployeeById(int employeeID)
             return await _context.Employees.FindAsync(employeeID);
         }
 
+        public async Task<DbEmployee> GetEmployeeById(string employeeID)
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/Repositories/EmployeeRepository.cs]: Name it `GetEmployeeByEmployeeId`, to make it explicit.

And can you look by non-key field?

**Review by tomasmuzas — APPROVED** [review]
Approved after minor changes :) Good job with error codes and exception handling!

**Review by mstruckus — COMMENTED** [review]
Ignore the mess in controller's method will fix that soon.

**Pull request merged**